src/Illuminate/Session/SessionManager.php

```diff
@@ -186,7 +186,12 @@ protected function buildSession($handler)
     {
         return $this->config->get('session.encrypt')
                 ? $this->buildEncryptedSession($handler)
-                : new Store($this->config->get('session.cookie'), $handler);
+                : new Store(
+                    $this->config->get('session.cookie'),
+                    $handler,
+                    $id = null,
```

Comment:
Is this assignment just to clarify the parameter name? If so, what about named parameters instead?